openmc/model/triso.py

```diff
@@ -1,13 +1,22 @@
+from __future__ import division
 import copy
-from collections import Iterable
+from collections import Iterable, defaultdict
 from numbers import Real
 import warnings
+import itertools
+import scipy.spatial
+from scipy.spatial.distance import cdist
```

Comment:
The SciPy imports should go down in the second block with numpy. Per [PEP8](https://www.python.org/dev/peps/pep-0008/#imports), the order is:

- Standard library imports
- Third-party imports
- local application imports

Each separated by a blank line